alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -131,8 +131,8 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
         val adminResponseMap = getAlerts(client(), inputMap, getHeader()).asMap()
         val expected = when (isHttps()) {
             true -> 4   // when test is run with security - get the correct filtered results.
-            false -> 0  // when test is run without security and filterby is enabled - fails to
-            // resolve user and filters by empty list, to get zero results.
+            false -> 4  // when test is run without security and filterby is enabled - filtering
+                        // does not work without security, so filtering is ignored and gets a result
```

Comment:
Even though the value is the same, I want to have it commented/formatted this way to explain whats going on.